site/content/en/docs/handbook/registry.md

```diff
@@ -42,7 +42,8 @@ You can use the `--insecure-registry` flag on the
 
 One nifty hack is to allow the kubelet running in minikube to talk to registries deployed inside a pod in the cluster without backing them
 with TLS certificates. Because the default service cluster IP is known to be available at 10.0.0.1, users can pull images from registries
-deployed inside the cluster by creating the cluster with `minikube start --insecure-registry "10.0.0.0/24"`.
+deployed inside the cluster by creating the cluster with `minikube start --insecure-registry "10.0.0.0/24"`. If using macOS, ensure the cluster
```

Comment:
lets remove "if using macOS" this would be same for all platforms

Reply:
Thanks, I wasn't sure if it was true on all operating systems